WtWSMS/common/landed_titles/BP_e_byzantium.txt

```diff
@@ -13988,44 +16409,12 @@ e_byzantium = {
 				}
 				b_vrestenitsa = {
 					province = 3704
-					
+
 					color = { 185 130 150 }
 				}
 				b_agrafa = {
 					province = 3705
-					cultural_names = {
-						#Roman Cultures
-						name_list_roman = cn_agrapha
-						name_list_romano_gallic = cn_agrapha
-						name_list_romano_hispanic = cn_agrapha
-						name_list_romano_british = cn_agrapha
-						name_list_romano_african = cn_agrapha
-						name_list_romano_raetian = cn_agrapha
-						name_list_romano_thracian = cn_agrapha
-						name_list_romano_illyrian = cn_agrapha
-						name_list_romano_pannonian = cn_agrapha
-						name_list_dalmatian = cn_agrapha
-						name_list_romano_noric = cn_agrapha
-						name_list_romano_gothic = cn_agrapha
-						name_list_romano_dacian = cn_agrapha
-						name_list_romano_frank = cn_agrapha
-						name_list_romano_aramean = cn_agrapha
-						name_list_romano_vandalic = cn_agrapha
-						#Byzantine Cultures
-						name_list_greek = cn_agrapha
-						name_list_aeolian = cn_agrapha
-						name_list_helleno_aramean = cn_agrapha
-						name_list_greek_gothic = cn_agrapha
-						name_list_greek_persian = cn_agrapha
-						name_list_isaurian_persian = cn_agrapha
-						name_list_grecco_axumite = cn_agrapha
-						name_list_bosporan = cn_agrapha
-						name_list_pontic = cn_agrapha
-						name_list_tsakonian = cn_agrapha
-						name_list_massalian = cn_agrapha
-						name_list_helleno_thracian = cn_agrapha
-						name_list_helleno_libyan = cn_agrapha
```

Comment:
Why the removal?

Reply:
> Why the removal?

That name was pretty anachronistic, but luckily I'm found better one; it is included in the latest commit.